src/npc/npc_kill.cpp

```diff
@@ -1471,7 +2083,24 @@ void KillNPC(int A, int B)
                 }
 
                 if(NPC[A].Type != 29 && NPC[A].Type != 47 && NPC[A].Type != 77)
-                    PlaySound(SFX_ShellHit);
+                    if(NPCScore[NPC[A].Type] == 1)
+                        PlaySound(SFX_ShellHit);
+                    else if(NPCScore[NPC[A].Type] == 2)
+                        PlaySound(SFX_ShellHit);
+                    else if(NPCScore[NPC[A].Type] == 3)
+                        PlaySound(SFX_Combo1);
+                    else if(NPCScore[NPC[A].Type] == 4)
+                        PlaySound(SFX_Combo2);
+                    else if(NPCScore[NPC[A].Type] == 5)
+                        PlaySound(SFX_Combo3);
+                    else if(NPCScore[NPC[A].Type] == 6)
+                        PlaySound(SFX_Combo4);
+                    else if(NPCScore[NPC[A].Type] == 7)
+                        PlaySound(SFX_Combo5);
+                    else if(NPCScore[NPC[A].Type] == 8)
+                        PlaySound(SFX_Combo6);
+                    else if(NPCScore[NPC[A].Type] >= 9)
+                        PlaySound(SFX_Combo7);
```

Comment:
A tip: such large blocks you MUST put into brackets:
```cpp
if(cond)
{
    // everything
}
```
However, please use the function that I made instead of this large stuff.